pkg/docker.go

```diff
@@ -0,0 +1,17 @@
+package pkg
+
+import (
+	"os"
+
+	"github.com/docker/docker/client"
+)
+
+func ListContainers() {
+	dockerSocketPath := os.Getenv("DOCKER_SOCKET_PATH")
```

Comment:
I had to use a different Docker context in order for the client to work.

Before that, I'd run into this error:
```
2024/08/27 16:47:21 Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```